client/src/screens/commons/Book/index.tsx

```diff
@@ -1045,6 +1041,22 @@ const CommonsBook = () => {
                                             {file.name}
                                           </a>
                                         )}
+                                        {
+                                          file.access === 'instructors' && user?.isAuthenticated && user?.verifiedInstructor && (
```

Comment:
If we're retrieving the non-public files as long as the user is authenticated (but not necessarily verified) it would be good to have an icon/warning for the non-verified users and to disable the download button in that case